src/shared/components/buttons.js

```diff
@@ -15,6 +15,7 @@ import { SvgIcon } from '@hypothesis/frontend-shared';
  * @prop {boolean} [pressed] - Is this button currently "active?" (set
  *   `aria-pressed`)
  * @prop {() => any} [onClick]
+ * @prop {Object} [style] - Optional inline styles
```

Comment:
In case we wanted to be more strict on types, this is something we could consider:
https://github.com/frenic/csstype

Reply:
Interesting to know about! My initial gut feeling is that the "expense" of having this as a dependency might outweigh its benefits for us, given the failure severity of using an invalid CSS property here (low).